doc/user_guide/Running_in_Webassembly.md

```diff
@@ -0,0 +1,176 @@
+
+```html
+<py-env>
+- panel
+...
+</py-env>
+```
+
+Once installed you will be able to `import panel` in your `<py-script>` tag. Again, make sure you also load Bokeh.js and Panel.js:
+
+
+```html
+<html>
+  <head>
+    <script type="text/javascript" src="https://cdn.bokeh.org/bokeh/release/bokeh-2.4.3.js"></script>
+    <script type="text/javascript" src="https://cdn.bokeh.org/bokeh/release/bokeh-widgets-2.4.3.min.js"></script>
+    <script type="text/javascript" src="https://cdn.bokeh.org/bokeh/release/bokeh-tables-2.4.3.min.js"></script>
+    <script type="text/javascript" src="https://cdn.jsdelivr.net/npm/@holoviz/panel@0.14.0/dist/panel.min.js"></script>
+
+    <link rel="stylesheet" href="https://pyscript.net/stable/pyscript.css" />
+    <script defer src="https://pyscript.net/stable/pyscript.js"></script>
+
+  </head>
+  <body>
+    <py-env>
+      - panel
+      ...
+    </py-env>
+    <py-script>
+      import panel as pn
+      ...
+    </py-script>
+  </body>
+</html>
+```
+
+## Rendering Panel components
+
+Rendering Panel components into the DOM is quite straightforward. You can simply use the `.servable()` method on any component and provide a target that should match the `id` of a DOM node:
+
+```python
+import panel as pn
+
+slider = pn.widgets.FloatSlider(start=0, end=10, name='Amplitude')
+
+def callback(new):
+    return f'Amplitude is: {new}'
+
+pn.Row(slider, pn.bind(callback, slider)).servable(target='simple_app');
+```
+
+This code will render this simple application into the `simple_app` DOM node:
+
+```python
+<div id="simple_app"></div>
+```
+
+Alternatively you can also use the `panel.io.pyodide.write` function to write into a particular DOM node:
+
+```python
+await pn.io.pyodide.write('simple_app', component)
+```
+
+### PyScript
+
+Current versions of PyScript will automatically render the output of the last cell of a <py-script> tag. E.g. in this example the `pn.Row()` component will be rendered wherever you placed the tag:
+
+```html
+<py-script>
+  import panel as pn
+
+  slider = pn.widgets.FloatSlider(start=0, end=10, name='Amplitude')
+
+  def callback(new):
+      return f'Amplitude is: {new}'
+
+  pn.Row(slider, pn.bind(callback, slider))
+</py-script>
+```
+
+## Converting Panel applications
+
+Writing an HTML file from scratch with all the Javascript and Python dependencies and other boilerplate can be quite annoying, and requires learning a good bit of HTML. To avoid writing all the boilerplate, Panel provides support for converting an entire application (including Panel templates) to an HTML file, using the `panel convert` API. As a starting point create one or more Python scripts or notebook files containing your application. The only requirement is that they import only global modules (relative imports of other scripts or modules is not supported).
```

Comment:
the 'global' modules still have to be on the [pyodide supported list](https://github.com/pyodide/pyodide/tree/main/packages) no? If so, maybe mention and add link.

Reply:
Good point, that's not quite accurate but I'll clarify.